### PR TITLE
Add krabsetw library to third_party

### PR DESCRIPTION
**Pull request by pierricgimmig**
"Krabs is a wrapper around ETW because ETW is the worst API ever made."

### CODE REVIEW

**Comment by pierricgimmig** [comment]
@beckerhe, @danielfenner , I didn't expect clang-format to run on third_party, is that normal?

**Comment by beckerhe** [comment]
> @beckerhe, @danielfenner , I didn't expect clang-format to run on third_party, is that normal?

Not intended. It's a bug I've seen before but forgot to fix. Let me quickly do that.

File: CMakeLists.txt [hunk 1]
```diff
@@ -109,7 +109,9 @@ endif()
 
 find_package(Filesystem REQUIRED)
 
-if(NOT WIN32)
+if(WIN32)
+  find_package(krabsetw REQUIRED)
```
Comment (beckerhe) [inline CMakeLists.txt:113]: Could you also add a line for copying the license file down at line 220?

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

**Comment by beckerhe** [comment]
A clang-format fix is here: #2733

**Review by danielfenner — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (pierricgimmig) [reply CMakeLists.txt:113]: Ah yes, added, thanks!

**Pull request merged**